Sphinx/README.md

```diff
@@ -1,19 +1,21 @@
 Missing:
```

Comment:
Can you make this file a little nicer to read? 

- Use Markdown headers
- Use [hyperlinks](http://yes)